spec/spec_helper.rb

```diff
@@ -20,7 +20,6 @@
 require 'rspec/its'
 require 'equivalent-xml/rspec_matchers'
 require 'logger'
-require 'pry' unless ENV['TRAVIS']
```

Comment:
we probably still want this, right? just `require 'pry'`, now?

Reply:
Or even just `unless ENV['CI']`